evalml/pipelines/regression/time_series_baseline_regression.py

```diff
@@ -1,64 +1,7 @@
-import numpy as np
-import pandas as pd
-
 from evalml.pipelines import TimeSeriesRegressionPipeline
-from evalml.utils.gen_utils import (
-    _convert_to_woodwork_structure,
-    _convert_woodwork_types_wrapper,
-    drop_rows_with_nans,
-    pad_with_nans
-)
 
 
 class TimeSeriesBaselineRegressionPipeline(TimeSeriesRegressionPipeline):
     """Baseline Pipeline for time series regression problems."""
     _name = "Time Series Baseline Regression Pipeline"
     component_graph = ["Time Series Baseline Regressor"]
-
-    def fit(self, X, y):
```

Comment:
This code was needed to accept the `X=None` case so we can delete it now! This means adding baselines for time series classification will be easy now since we can just reuse the time series baseline regressor and not worry about implementing pipeline `fit` or `predict` !

Reply:
Ooh that's great!